src/compile/format.ts

```diff
@@ -161,8 +179,26 @@ export function guideFormat(
 ) {
   if (isCustomFormatType(formatType)) {
     return undefined; // handled in encode block
-  } else if (format === undefined && formatType === undefined && config.numberFormatType && config.customFormatTypes) {
-    return undefined; // handled in encode block
+  } else if (format === undefined && formatType === undefined && config.customFormatTypes) {
+    if (
+      config.normalizedNumberFormatType &&
+      isPositionFieldOrDatumDef(fieldOrDatumDef) &&
+      fieldOrDatumDef.stack === 'normalize' &&
+      channelDefType(fieldOrDatumDef) === 'quantitative'
```

Comment:
I now wonder if the existing code forgot to apply "      `channelDefType(fieldOrDatumDef) === 'quantitative'`" too. 

Basically should we move `channelDefType(fieldOrDatumDef) === 'quantitative'` to `if (format === undefined && formatType === undefined && config.customFormatTypes)` above?